pkg/kubelet/network/kubenet/kubenet_linux.go

```diff
@@ -334,6 +334,9 @@ func (plugin *kubenetNetworkPlugin) Capabilities() utilsets.Int {
 	return utilsets.NewInt(network.NET_PLUGIN_CAPABILITY_SHAPING)
 }
 
+// setup sets up networking through CNI using the given ns/name and sandbox ID.
+// TODO: Don't pass the pod to this method, it only needs it for bw shaping and
```

Comment:
tiny nit: expand `bw` to `bandwidth` to help readers less familiar with the code.